extension/src/helpers/metrics.ts

```diff
@@ -30,17 +29,6 @@ export function metricsMiddleware<State>(): Middleware<object, State> {
     };
 }
 
-// I can't figure out how to get the properties off a thunk for the ActionType
-// without creating an intermediate value
-// eslint-disable-next-line
-const dummyThunk = createAsyncThunk<any, any>("dummy", () => {});
-const dummyAction = createAction<any>("also dummy");
-type ActionType =
-  | typeof dummyThunk.fulfilled
-  | typeof dummyThunk.rejected
-  | typeof dummyThunk.pending
-  | typeof dummyAction;
-
```

Comment:
this doesn't play well with the newer version of redux toolkit. This was pretty hacky anyway, I don't think we need to work this hard to deal with redux toolkit type weirdness